invalidate-cache.ts

```diff
@@ -0,0 +1,7 @@
+const { execSync } = require('child_process');
+const stackName = 'AwsSdkStack';
+const outputKey = 'DistributionId';
+
+const distributionId = execSync(`aws cloudformation describe-stacks --stack-name ${stackName} --query "Stacks[0].Outputs[?OutputKey=='${outputKey}'].OutputValue" --output text`).toString().trim();
+
+execSync(`aws cloudfront create-invalidation --distribution-id ${distributionId} --paths '/*'`, { stdio: 'inherit' });
```

Comment:
Could you please explain why this code is needed? I guess the following lines should make invalidations as well, or maybe I just don't get it

```
  new s3Deployment.BucketDeployment(this, 'DeployWebsite', {
        sources: [s3Deployment.Source.asset('../dist')],
        destinationBucket: bucket,
        distribution,
        distributionPaths: ['/*'],
      });
 ```

Reply:
I updated the code according to your comments